src/screens/ContactScreen/elements/RenderItem.js

```diff
@@ -24,7 +24,7 @@ const RenderItem = ({item, selectedUsers, onChange, index}) => {
           // backgroundColor:
           //   isInArray(item, selectedUsers) > -1
           //     ? 'rgba(0, 173, 181, 0.15)'
-          //     : '#FFFFFF',
+          //     : COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `'#FFFFFF'` to `COLORS.white` is a good move towards standardizing color usage across the application. However, it seems like this line of code is commented out. If this code isn't needed, consider removing it to improve readability and maintainability.
```diff
-           //     : '#FFFFFF',
+           //     : COLORS.white,
```
If this code is temporarily commented out for testing or debugging purposes, remember to uncomment it when you're done.


<!-- This is an auto-generated comment by OSS CodeRabbit -->